docs/source/crawler/basics.rst

```diff
@@ -27,5 +27,22 @@ a modified dictionary as its output once.
    or modified.  To help with creation of a new shallow copy with some fields
    adjusted, use :func:`~datalad.utils.updated`.
 
-Pipeline
---------
\ No newline at end of file
+Pipelines
+---------
+
+A pipeline is a series of functions gathered into a list. Each function take the
```

Comment:
good, but remember that those aren't functions, but generators ;)  and they might be defined as independent functions OR as generator methods of a class/instance, like those you use below -- crawl_url is actually a class name